java/garbage-collector/src/main/java/sleeper/garbagecollector/FileToDelete.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022-2025 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.garbagecollector;
+
+import java.util.Set;
+import java.util.stream.Stream;
+
+public record FileToDelete(String filename, String bucketName, String objectKey) {
```

Comment:
Can we have some Javadoc for the class, please?

Reply:
Done